src/spaces/test/spaces_test.cpp

```diff
@@ -1,50 +1,47 @@
 #include "src/spaces/spaces.hpp"
 
-#include <gtest/gtest.h>
+#include "boost/ut.hpp"
 
 // NOLINTBEGIN(readability-magic-numbers)
 
-namespace spaces {
+using vec = spaces::Vector<float, 3>;
+using point = spaces::Point<float, 3>;
 
-namespace test {}
-
-using vec = Vector<float, 3>;
-using point = Point<float, 3>;
-
-void spaces_vector()
+auto main() -> int
 {
-    constexpr vec zero{};
-    constexpr vec v1{1.0F, 0.0F, 0.0F};
-    constexpr vec v2{1.0F, 0.0F, -1.0F};
+    using namespace boost::ut;
 
-    static_assert(v1.get<0>() == 1.0F);
-    static_assert(v1.get<1>() == 0.0F);
-    static_assert(v1.get<2>() == 0.0F);
+    test("spaces vector") = [] {
+        constexpr vec zero{};
+        constexpr vec v1{1.0F, 0.0F, 0.0F};
+        constexpr vec v2{1.0F, 0.0F, -1.0F};
 
-    static_assert(v1 + zero == v1);
+        static_assert(v1.get<0>() == 1.0F);
+        static_assert(v1.get<1>() == 0.0F);
+        static_assert(v1.get<2>() == 0.0F);
 
-    static_assert(-v1 == vec{-1.0F, 0.0F, 0.0F});
-    static_assert(v1 + v2 == vec{2.0F, 0.0F, -1.0F});
+        static_assert(v1 + zero == v1);
```

Comment:
You can try something like this to get better error messages when writing the tests:
```
constexpr auto expr = eq(v1 + zero, v1);
expect(expr);
```